lib/collection/src/collection_state.rs

```diff
@@ -7,12 +7,13 @@ use crate::collection::Collection;
 use crate::config::CollectionConfig;
 use crate::operations::types::CollectionResult;
 use crate::shard::remote_shard::RemoteShard;
-use crate::shard::{create_shard_dir, ChannelService, PeerId, Shard, ShardId};
+use crate::shard::{create_shard_dir, ChannelService, PeerId, Shard, ShardId, ShardTransfer};
 
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
 pub struct State {
     pub config: CollectionConfig,
     pub shard_to_peer: HashMap<ShardId, PeerId>,
+    pub transfers: HashSet<ShardTransfer>,
```

Comment:
will it be backward compatible?

Reply:
We don't store snapshots, so in terms of loading - yes. But it won't be possible to run 2 nodes of different versions - one before this commit, one after.

Reply:
But I don't think we plan to support that later case, do we?